dags/github/neo4j_storage/comments.py

```diff
@@ -91,6 +108,19 @@ def save_comment_to_neo4j(comment: dict, repository_id: str):
     else:
         user_query = ""
 
+    if reactions_member:
+        reactions_member_query = f"""
+            WITH c
+            UNWIND $reactions_member as reaction_member
+            MERGE (ghu:{Node.GitHubUser.value} {{id: reaction_member.user.id}})
+                SET ghu += reaction_member.user, ghu.latestSavedAt = datetime()
+            WITH c, ghu
+            MERGE (c)-[ra:{Relationship.REACTED.value}]->(ghu)
```

Comment:
The same my previous comment. We want the github user to react on github comment.